.github/workflows/helm.yml

```diff
@@ -0,0 +1,28 @@
+name: Build & Push Chart
+
+on: push
+#  push:
+#    branches:
+#      - main
+    
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+      - name: Helm lint & package & push
+        run: |
+          cd chart
+          helm registry login ${GHR} -u ${{ github.actor }} -p ${{ secrets.GITHUB_TOKEN }}
+          helm registry login ${MTR} -u ${DOCKER_USERNAME} -p ${DOCKER_PASSWORD}
+          helm lint .
+          helm package .
+          helm push $(ls *.tgz| head -1) oci://${GHR}/${{ github.repository_owner }}/charts
+          helm push $(ls *.tgz| head -1) oci://${MTR}/${ORG}/charts
```

Comment:
Why do we want to push it to the MTR? I think the ghcr is enough since this is an open source project.

Reply:
removed and mv to release action